src/paperless/settings.py

```diff
@@ -90,9 +95,10 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "..", "data", "db.sqlite3"),
+        "NAME": os.path.join(os.getenv("PAPERLESS_DBLOCATION",os.path.join(BASE_DIR, "..", "data")), "db.sqlite3")
```

Comment:
Nitpick: you've got a couple of pep8 violations here:

* The line is too long (should be less than 80 characters)
* There's no space between `,` and `os.`.  Maybe something like this instead?

```python
os.path.join(
    os.getenv(
        "PAPERLESS_DBLOCATION",
        os.path.join(BASE_DIR, "..", "data")
    ), 
    "db.sqlite3"
)
```

Additionally, if `PAPERLESS_DBLOCATION` is set to the value of the database *file* wouldn't you end up with something like `/path/to/db.sqlite3/db.sqlite3`?

Maybe this would be better?

```python
os.getenv(
    "PAPERLESS_DBLOCATION",
    os.path.join(BASE_DIR, "..", "data", "db.sqlite3")
)
```

Unless of course you meant to have `PAPERLESS_DBLOCATION` set to just the directory?  In which perhaps the variable name should be `PAPERLESS_DB_DIRECTORY` just so it's obvious?